cpp/ql/lib/semmle/code/cpp/security/FlowSources.qll

```diff
@@ -53,7 +53,7 @@ private class ArgvSource extends LocalFlowSource {
     exists(Function main, Parameter argv |
       main.hasGlobalName("main") and
       main.getParameter(1) = argv and
-      argv.getAnAccess() in [this.asExpr(), this.asIndirectExpr()]
```

Comment:
I _think_ we should have a change note for this.

Reply:
I think so too.

Reply:
Change note added.